Intermittent issue building Krita (4.0.x)

When I try to build krita I intermittently get the following build failure:
`/tmp/nix-build-krita-4.0.0-pre2c.drv-0/krita-1b1695a/plugins/extensions/pykrita/plugin/pyqtpluginsettings.cpp:19:24: fatal error: ui_manager.h: No such file or directory #include "ui_manager.h"`

By intermittent I mean if it builds it builds, but sometimes if fails with that message and then I don't know how to fix it. I think sometimes running garbage collection helps, but I'm not sure.

I run it with: `nix-shell -p 'libsForQt5.callPackage ./default.nix {openjpeg = openjpeg_1;}'
 --cores 4`

I just use a slightly modified version of the nix expression in nixpkgs: 
```nix
{ fetchgit, python36, python36Packages, stdenv, lib, fetchurl, fetchpatch, cmake, extra-cmake-modules
, karchive, kconfig, kwidgetsaddons, kcompletion, kcoreaddons
, kguiaddons, ki18n, kitemmodels, kitemviews, kwindowsystem
, kio, kcrash
, boost, libraw, fftw, eigen, exiv2, lcms2, gsl, openexr
, openjpeg, opencolorio, vc, poppler_qt5, curl, ilmbase
, qtmultimedia, qtx11extras
}:

stdenv.mkDerivation rec {
  name = "krita-${version}";
  version = "4.0.0-pre2c";

  cmakeFlags = ["-DPYQT_SIP_DIR_OVERRIDE=${python36Packages.pyqt5.outPath}/share/sip/PyQt5" "-DBUILD_TESTING=TRUE" "-DHIDE_SAFE_ASSERTS=FALSE" "-DCMAKE_BUILD_TYPE=RelWithDebInfo" ];

  src = fetchgit {
    url = "git://github.com/KDE/krita.git";
    rev = "1b1695a99d9417cd2e84465ec56ffc2636de0b59";
    sha256 = "1madxhfmvcazvm90s4hgwg0hc78f778ln7rmmswyn6aq2gygvp2p";
  };

  nativeBuildInputs = [ cmake extra-cmake-modules ];

  propagatedBuildInputs = [python36 python36Packages.pyqt5 python36Packages.sip];

  buildInputs = [
    karchive kconfig kwidgetsaddons kcompletion kcoreaddons kguiaddons
    ki18n kitemmodels kitemviews kwindowsystem kio kcrash
    boost libraw fftw eigen exiv2 lcms2 gsl openexr
    openjpeg opencolorio vc poppler_qt5 curl ilmbase
    qtmultimedia qtx11extras
  ];

  enableParallelBuilding = true;
  NIX_CFLAGS_COMPILE = [ "-I${ilmbase.dev}/include/OpenEXR" ];

  meta = with lib; {
    description = "A free an open source painting application";
    homepage = https://krita.org/;
    maintainers = with maintainers; [ abbradar ];
    platforms = platforms.linux;
    license = licenses.gpl2;
  };
}
```

Here's my nix-info stuff:

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.9.81, NixOS, 17.09.3023.c882c4dd55e (Hummingbird)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
 - channels(root): `"nixos-17.09.3023.c882c4dd55e"`
 - channels(user): `"unstable-18.03pre127302.e860b651d6e"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`
```

--- a/pkgs/applications/graphics/krita/default.nix
+++ b/pkgs/applications/graphics/krita/default.nix
@@ -10,15 +10,12 @@
 
 mkDerivation rec {
   pname = "krita";
-  version = "4.4.1";
+  version = "4.4.2";
 
   src = fetchurl {
     url = "https://download.kde.org/stable/${pname}/${version}/${pname}-${version}.tar.xz";
-    sha256 = "1bmmfvmawnlihbqkksdrwxfkaip4nfsi97w83fmvkyxl4jk715vr";
+    sha256 = "121fjdv5phx1aqk21vx9k9vsc5k1w8s86gp6pamy2y31r2ph7r8y";
   };
-
-  # *somtimes* fails with can't find ui_manager.h, also see https://github.com/NixOS/nixpkgs/issues/35359
-  enableParallelBuilding = false;
 
   nativeBuildInputs = [ cmake extra-cmake-modules python3Packages.sip makeWrapper ];
 

